tests/operator_test.go

```diff
@@ -466,9 +466,8 @@ var _ = Describe("ALL Operator tests", func() {
 				Expect(pvc).ToNot(BeNil())
 
 				By("Verify dv succeeded")
-				dv, err := f.CdiClient.CdiV1beta1().DataVolumes(pvc.Namespace).Get(context.TODO(), pvc.Name, metav1.GetOptions{})
+				err = utils.WaitForDataVolumePhase(f, pvc.Namespace, cdiv1.Succeeded, pvc.Name)
```

Comment:
Might make more sense to use `WaitForDataVolumeGC` instead. Not sure. This function will call that eventually due to `Succeeded`, but my first question that popped up was, isn't the DV GCed at this point?

Reply:
WaitForDataVolumePhase is prefarable as it is agostic to wheter GC is enabled or not.

Reply:
`WaitForDataVolumePhase/WithTimeout(...cdiv1.Succeeded...)` is refered 62 times all around the func tests. I don't think the suggested rename worth it this time.

Reply:
So we have 62 locations that are confusing when GC is enabled? Could we at the very least put a comment on the function indicating that it also accounts for the DV being garbage collected if the passed in phase is succeeded? That way when people look at the function it kind of makes sense.

Reply:
Comment added